src/compute-engine/library/arithmetic.ts

```diff
@@ -830,8 +829,7 @@ export const ARITHMETIC_LIBRARY: IdentifierDefinitions[] = [
         const [base, exp] = args;
         return canonicalRoot(base, exp);
       },
-      evaluate: ([x, n], { numericApproximation }) =>
-        root(x, n, { numericApproximation }),
+      evaluate: ([x, n], { numericApproximation }) => x.root(n),
```

Comment:
That doesn't sound correct: it ignores the numericApproximation flag. The previous implementation looks good.

Reply:
I don't remember making this change because I didn't look that the evaluate stuff. I believe this may be an issue with Github.